lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
       "public gem pushes."
   end
 
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
+  spec.files         = Dir.chdir(__dir__){`git ls-files -z`}.split("\x0").reject do |f|
```

Comment:
let's use `File.expand_path("..", __FILE__)` for compatibility, and maybe add some spacing to conform with the style in the rest of the codebase?

Reply:
Done ✅, also moved the whole line inside the chdir block to make it more readable